### PR TITLE
[wavpack] arm support

### PR DESCRIPTION
**Pull request by daschuer**
This enables arm builds of the wavpack port

- #### What does your PR fix?  
  Wrong CMAKE_SYSTEM_PROCESSOR on Osx and missing asm compiler definition of Linux

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  In addition linux-arm and crosscompiling for Apple silicon 

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  `Yes`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  `Yes`

### CODE REVIEW

**Comment by JackBoosY** [comment]
The regressions are not related to this changes.

**Comment by daschuer** [comment]
The Ci failure is unrelated. See: https://github.com/microsoft/vcpkg/issues/21758

**Comment by JackBoosY** [comment]
There also have a `chromium-base:x64-osx` regression.

**Comment by daschuer** [comment]
I cannot confirm the chromium-base:x64-osx failure in my repository:
https://github.com/daschuer/vcpkg/runs/4428747869?check_suite_focus=true

Do you have an idea how to solve the issues. Are there anywhere the detailed log files of the failure? 
Other ideas?

**Comment by JackBoosY** [comment]
```
FAILED: obj/base/base/superfasthash.o 
../../../../../../usr/bin/clang -MMD -MF obj/base/base/superfasthash.o.d -DSYSTEM_NATIVE_UTF8 -D_LIBCPP_HAS_NO_ALIGNED_ALLOCATION -DCR_XCODE_VERSION= -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__ASSERT_MACROS_DEFINE_VERSIONS_WITHOUT_UNDERSCORES=0 -D_DEBUG -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DBASE_IMPLEMENTATION -I../src/25ce732 -Igen -fno-strict-aliasing -fstack-protector-strong -fcolor-diagnostics -fmerge-all-constants -fcrash-diagnostics-dir=../src/25ce732/tools/clang/crashreports -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -arch x86_64 -Wno-builtin-macro-redefined -D__DATE__= -D__TIME__= -D__TIMESTAMP__= -no-canonical-prefixes -Wall -Wextra -Wimplicit-fallthrough -Wunreachable-code -Wthread-safety -Wextra-semi -Wunguarded-availability -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-ignored-pragma-optimize -Wno-implicit-int-float-conversion -Wno-final-dtor-non-final-class -Wno-builtin-assume-aligned-alignment -Wno-deprecated-copy -Wno-non-c-typedef-for-linkage -Wmax-tokens -O0 -fno-omit-frame-pointer -fno-standalone-debug -gdwarf-4 -g2 -isysroot ../../../../../../Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -mmacosx-version-min=10.10.0 -ftrivial-auto-var-init=pattern -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wno-unknown-warning-option -Wno-unknown-pragmas -Wglobal-constructors -Wexit-time-destructors -Wshadow -Wexit-time-destructors -std=c11 -Wno-implicit-fallthrough -include obj/base/base/precompile.h-c -c ../src/25ce732/base/third_party/superfasthash/superfasthash.c -o obj/base/base/superfasthash.o
/bin/sh: ../../../../../../usr/bin/clang: No such file or directory
```
Should not related to your changes.

**Comment by BillyONeal** [comment]
Can you try doing a merge or rebase with master to see if that clears up the failure? The merge-base here is really old. I tried to push such a merge for you but the PR was created without the "allow edits from maintainers" option checked so I can't help

**Comment by daschuer** [comment]
Rebased. 

@JackBoosY can we see the posted lines somewhere at Azure?

**Comment by JackBoosY** [comment]
@daschuer See https://dev.azure.com/vcpkg/public/_build/results?buildId=64249&view=results.
Or click `Details` in the pipeline test area then click `View more details on Azure Pipelines`.

**Comment by daschuer** [comment]
The old issue is gone, but new have popped up:
```
##[error]REGRESSION: tensorflow:x64-osx. If expected, add tensorflow:x64-osx=fail to .\scripts\ci.baseline.txt.
##[error]REGRESSION: tensorflow-cc:x64-osx. If expected, add tensorflow-cc:x64-osx=fail to .\scripts\ci.baseline.txt. 
##[error]REGRESSION: qtapplicationmanager:x64-linux. If expected, add qtapplicationmanager:x64-linux=fail to .\scripts\ci.baseline.txt.
```

**Comment by daschuer** [comment]
```
/mnt/vcpkg-ci/buildtrees/qtapplicationmanager/src/ad43fc4b04-cc8e3e67d5.clean/src/common-lib/qtyaml.cpp:39:10: fatal error: yaml.h: No such file or directory
```
Probably a regression from https://github.com/microsoft/vcpkg/pull/21818 ?

**Comment by BillyONeal** [comment]
Should be fixed by https://github.com/microsoft/vcpkg/pull/21925

**Comment by BillyONeal** [comment]
(Can you merge with 'master'?)

**Comment by JonLiu1993** [comment]
@daschuer ,Plesase merge with 'master'
````
remote: Permission to daschuer/vcpkg.git denied to JonLiu1993
````

**Comment by JackBoosY** [comment]
Depends on https://github.com/microsoft/vcpkg/pull/21912.

**Comment by daschuer** [comment]
The x64_linux issue is an unrelated regression from: 
https://github.com/microsoft/vcpkg/pull/21565

**Comment by JackBoosY** [comment]
The mpg123 regression will be fixed by https://github.com/microsoft/vcpkg/pull/22028.

**Comment by daschuer** [comment]
For some reasons this PR is a regression detector .. 
Now 
```activemq-cpp:x64-linux: BUILD_FAILED: 44.07```
```
/usr/bin/ld: cannot find -luuid
```

**Comment by JackBoosY** [comment]
@daschuer It seems that Wavpack is implicitly dependent on libuuid through a certain option.
So please disable this option or add libuuid as a dependency / feature of wavpack and modify the find / usage method.

**Comment by daschuer** [comment]
It was the activemq-cpp that was failing unrelated due to e missing dependency, not the wavpack package. 

I have issues a PR that should solve the issue: 
https://github.com/microsoft/vcpkg/pull/22059

**Comment by JackBoosY** [comment]
@daschuer Ah that's correct! Thanks for your PR.

**Comment by JackBoosY** [comment]
The regressions will be fixed by https://github.com/microsoft/vcpkg/pull/22110.

**Comment by BillyONeal** [comment]
Can you either merge with `master` or let maintainers edit the PR so that we can do that for you? Thanks!

**Comment by daschuer** [comment]
Both done.

**Comment by JackBoosY** [comment]
Will rerun the osx test after others finish.

**Comment by daschuer** [comment]
The regression detector is still working ;-) 

Now it has found:
REGRESSION: llvm:x64-osx. If expected, add llvm:x64-osx=fail to .\scripts\ci.baseline.txt.
Undefined symbols for architecture x86_64: "__nc_panelhook_sp"

**Comment by BillyONeal** [comment]
> Can you either merge with `master` or let maintainers edit the PR so that we can do that for you? Thanks!

I think @JackBoosY fixed it, can you do this again?

**Review by JackBoosY — APPROVED** [review]
I'd like other guys to review the toolchain part.

**Review by BillyONeal — APPROVED** [review]

File: scripts/toolchains/osx.cmake [hunk 1]
```diff
@@ -2,17 +2,26 @@ if(NOT _VCPKG_OSX_TOOLCHAIN)
...
+if(VCPKG_TARGET_ARCHITECTURE STREQUAL "x64")
+   set(CMAKE_SYSTEM_PROCESSOR x86_64 CACHE STRING "")
+elseif(VCPKG_TARGET_ARCHITECTURE STREQUAL "x86")
+   set(CMAKE_SYSTEM_PROCESSOR x86 CACHE STRING "")
+elseif(VCPKG_TARGET_ARCHITECTURE STREQUAL "arm64")
+   set(CMAKE_SYSTEM_PROCESSOR arm64 CACHE STRING "")
+elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL "Darwin")
+   set(CMAKE_SYSTEM_PROCESSOR "${CMAKE_HOST_SYSTEM_PROCESSOR}" CACHE STRING "")
+else()
+   set(CMAKE_SYSTEM_PROCESSOR x86_64 CACHE STRING "")
```
Comment (strega-nil-ms) [inline scripts/toolchains/osx.cmake:22]: This is an error; we do not support anything other than x86_64 and ARM64 on macOS (sorry powerbook fans), and `VCPKG_TARGET_ARCHITECTURE` must be set correctly

```suggestion
else()
    message(FATAL_ERROR "Unrecognized VCPKG_TARGET_ARCHITECTURE for macOS - ${VCPKG_TARGET_ARCHITECTURE}")
```

**Review by strega-nil-ms — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Thank you @strega-nil-ms 
Done

**Comment by daschuer** [comment]
Unfortunately glib and urho3d fails now:
glib:
```../src/glib-2-3e22263b95.clean/meson.build:2051:0: ERROR: Dependency Intl not found: Unable to obtain CMake system information
```
urho3d:
```
Undefined symbols for architecture x86_64:
  "_BC", referenced from:
      __rl_init_terminal_io in libreadline.a(terminal.c.o)
```

**Review by strega-nil-ms — APPROVED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
I can confirm this changes breaks the glib:x64-oxs build:
```
7991 Try CMake generator: Unix Makefiles
7992 Calling CMake (['/Users/vcpkg/Documents/vcpkg/21790/vcpkg/downloads/tools/cmake-3.21.1-osx/cmake-3.21.1-macos-universal/CMake.app/Contents/bin/cmake']) in /Users/vcpkg/Documents/vcpkg/21790/vcpkg/buildtrees/glib/x64-osx-dbg/meson-private/cmake_Intl with:
7993   - "-G"
7994   - "Unix Makefiles"
7995   - "--trace-expand"
7996   - "--trace-format=json-v1"
7997   - "--no-warn-unused-cli"
7998   - "--trace-redirect=cmake_trace.txt"
7999   - "-DCMAKE_TOOLCHAIN_FILE=/Users/vcpkg/Documents/vcpkg/21790/vcpkg/buildtrees/glib/x64-osx-dbg/meson-private/cmake_Intl/CMakeMesonToolchainFile.cmake"
8000   - "."
8001   - "-DCMAKE_PREFIX_PATH=/Users/vcpkg/Documents/vcpkg/21790/vcpkg/installed/x64-osx/debug;/Users/vcpkg/Documents/vcpkg/21790/vcpkg/installed/x64-osx"
8002 CMake failed to gather system information for generator Unix Makefiles with error code 1
8003 OUT:
8004 Running with expanded trace output on.
8005 Not searching for unused variables given on the command line.
8006 Trace will be written to cmake_trace.txt
8007 -- Configuring incomplete, errors occurred!
8008
8009
8010
8011 ERR:
8012 CMake Error at /Users/vcpkg/Documents/vcpkg/21790/vcpkg/scripts/toolchains/osx.cmake:19 (message):
8013   Unrecognized VCPKG_TARGET_ARCHITECTURE for macOS -
8014 Call Stack (most recent call first):
8015   /Users/vcpkg/Documents/vcpkg/21790/vcpkg/scripts/buildsystems/vcpkg.cmake:215 (include)
8016   CMakeMesonToolchainFile.cmake:115 (include)
8017   /Users/vcpkg/Documents/vcpkg/21790/vcpkg/downloads/tools/cmake-3.21.1-osx/cmake-3.21.1-macos-universal/CMake.app/Contents/share/cmake-3.21/Modules/CMakeDetermineSystem.cmake:124 (include)
8018   CMakeLists.txt:3 (project)
8019
8020
8021 CMake Error: CMake was unable to find a build program corresponding to "Unix Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.

```

**Comment by daschuer** [comment]
OK I have reverted the change suggested here https://github.com/microsoft/vcpkg/pull/21790#discussion_r776854987, to fix the build. 

It turns out that the osx.cmake file is used two times one time in the vcpkg context with no CMAKE variables set and one time in the meson context where the info form the vcpkg triplet already have been converted to CMAKE variables and the triplet file content is not available.  
This means setting the CMAKE_SYSTEM_PROCESSOR via CMAKE_HOST_SYSTEM_PROCESSOR is correct in the later case.

This PR fixes the vcpkg run, where CMAKE_SYSTEM_PROCESSOR was always set to "x86_64" unconditionally which is not correct and fails on "amr64"

File: ports/wavpack/portfile.cmake [hunk 2]
```diff
@@ -1,4 +1,6 @@
-vcpkg_fail_port_install(ON_ARCH "arm" "arm64")
+if(VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_fail_port_install(ON_ARCH "arm" "arm64")
+endif()
```
Comment (dg0yt) [inline ports/wavpack/portfile.cmake:3]: Remember to remove this block. It is already achieved by `"supports": "!(windows & arm)"` in `vcpkg.json.

**Review by JackBoosY — APPROVED** [review]

**Review by strega-nil-ms — CHANGES_REQUESTED** [review]
The macOS thing needs to be fixed; we do not support anything other than x86, x64, and arm64 on macOS, and this should be correctly done in the toolchain

**Comment by daschuer** [comment]
@strega-nil-ms I am not sure what is still incorrect. Please file a proposal that shows the changes. 

At least osx.cmake should be correct now as explained here: https://github.com/microsoft/vcpkg/pull/21790#issuecomment-1005210085

**Comment by daschuer** [comment]
@strega-nil-ms Is this mergwable now?

**Comment by BillyONeal** [comment]
> @strega-nil-ms I am not sure what is still incorrect. Please file a proposal that shows the changes.
> 
> At least osx.cmake should be correct now as explained here: [#21790 (comment)](https://github.com/microsoft/vcpkg/pull/21790#issuecomment-1005210085)

This comment remains unaddressed: https://github.com/microsoft/vcpkg/pull/21790/files#r776854987

**Comment by BillyONeal** [comment]
>and one time in the meson context where the info form the vcpkg triplet already have been converted to CMAKE variables and the triplet file content is not available.

That is broken; the triplet needs to control Meson's behavior.

**Comment by daschuer** [comment]
> This comment remains unaddressed: https://github.com/microsoft/vcpkg/pull/21790/files#r776854987

This was addressed. It tuns out that the assumption was wrong and the build fails after that. I had to revert the changes and drilled it down to the roots. The conclusion is described here: https://github.com/microsoft/vcpkg/pull/21790#issuecomment-1005210085

> That is broken; the triplet needs to control Meson's behavior.

So the assumption remains correct "the triplet needs to control Meson's behavior."
It works like that: The triplet setting is used to configures a cmake project used to create the  VCPKG_DETECTED_CMAKE_* variables. This cmake project used the osx.cmake file as toolchain file, but does not have access to the original variables from the triplet.   

So we have this dataflow: Triplet -> cmake -> meson.

File: scripts/toolchains/osx.cmake (hunk 1 above)
Comment (daschuer) [reply scripts/toolchains/osx.cmake:22]: Building fails with this change, see: 
https://github.com/microsoft/vcpkg/pull/21790#issuecomment-1005210085

**Comment by strega-nil** [comment]
Mmh, alright, could you do this instead then? I don't want to default to x86_64:

```cmake
if(DEFINED CMAKE_SYSTEM_PROCESSOR)
elseif(VCPKG_TARGET_ARCHITECTURE STREQUAL "x64")
   set(CMAKE_SYSTEM_PROCESSOR x86_64 CACHE STRING "")
elseif(VCPKG_TARGET_ARCHITECTURE STREQUAL "x86")
   set(CMAKE_SYSTEM_PROCESSOR x86 CACHE STRING "")
elseif(VCPKG_TARGET_ARCHITECTURE STREQUAL "arm64")
   set(CMAKE_SYSTEM_PROCESSOR arm64 CACHE STRING "")
else()
   set(CMAKE_SYSTEM_PROCESSOR "${CMAKE_HOST_SYSTEM_PROCESSOR}" CACHE STRING "")
endif()
```

this should work for all cases.

**Comment by daschuer** [comment]
Done. This work for me as well here: https://github.com/daschuer/vcpkg/actions/runs/1753511087

**Comment by daschuer** [comment]
This is still failing: 
```
FAILED: obj/base/base/precompile.h-mm.gch 
../../../../../../usr/bin/clang++ -MMD -MF obj/base/base/precompile.h-mm.gch.d -DSYSTEM_NATIVE_UTF8 -D_LIBCPP_HAS_NO_ALIGNED_ALLOCATION -DCR_XCODE_VERSION= -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__ASSERT_MACROS_DEFINE_VERSIONS_WITHOUT_UNDERSCORES=0 -D_DEBUG -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DBASE_IMPLEMENTATION -I../src/25ce732 -Igen  -fno-strict-aliasing -fstack-protector-strong -fcolor-diagnostics -fmerge-all-constants -fcrash-diagnostics-dir=../src/25ce732/tools/clang/crashreports -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -arch x86_64 -Wno-builtin-macro-redefined -D__DATE__= -D__TIME__= -D__TIMESTAMP__= -no-canonical-prefixes -Wall -Wextra -Wimplicit-fallthrough -Wunreachable-code -Wthread-safety -Wextra-semi -Wunguarded-availability -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-ignored-pragma-optimize -Wno-implicit-int-float-conversion -Wno-final-dtor-non-final-class -Wno-builtin-assume-aligned-alignment -Wno-deprecated-copy -Wno-non-c-typedef-for-linkage -Wmax-tokens -O0 -fno-omit-frame-pointer -fno-standalone-debug -gdwarf-4 -g2 -isysroot ../../../../../../Library/Developer/CommandLineTools/SDKs/MacOSX11.3.sdk -mmacosx-version-min=10.10.0 -ftrivial-auto-var-init=pattern -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wno-unknown-warning-option -Wno-unknown-pragmas -Wglobal-constructors -Wexit-time-destructors -Wshadow -Wexit-time-destructors -Wno-undefined-bool-conversion -Wno-tautological-undefined-compare -std=c++14 -fno-trigraphs -Wno-trigraphs -stdlib=libc++ -fobjc-call-cxx-cdtors -Wobjc-missing-property-synthesis -fno-exceptions -fno-rtti -fvisibility-inlines-hidden -x objective-c++-header -c ../src/25ce732/build/precompile.h -o obj/base/base/precompile.h-mm.gch
/bin/sh: ../../../../../../usr/bin/clang++: No such file or directory
[41/412] CXX obj/base/base/precompile.h-cc.gch
FAILED: obj/base/base/precompile.h-cc.gch 
../../../../../../usr/bin/clang++ -MMD -MF obj/base/base/precompile.h-cc.gch.d -DSYSTEM_NATIVE_UTF8 -D_LIBCPP_HAS_NO_ALIGNED_ALLOCATION -DCR_XCODE_VERSION= -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__ASSERT_MACROS_DEFINE_VERSIONS_WITHOUT_UNDERSCORES=0 -D_DEBUG -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DBASE_IMPLEMENTATION -I../src/25ce732 -Igen -fno-strict-aliasing -fstack-protector-strong -fcolor-diagnostics -fmerge-all-constants -fcrash-diagnostics-dir=../src/25ce732/tools/clang/crashreports -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -arch x86_64 -Wno-builtin-macro-redefined -D__DATE__= -D__TIME__= -D__TIMESTAMP__= -no-canonical-prefixes -Wall -Wextra -Wimplicit-fallthrough -Wunreachable-code -Wthread-safety -Wextra-semi -Wunguarded-availability -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-ignored-pragma-optimize -Wno-implicit-int-float-conversion -Wno-final-dtor-non-final-class -Wno-builtin-assume-aligned-alignment -Wno-deprecated-copy -Wno-non-c-typedef-for-linkage -Wmax-tokens -O0 -fno-omit-frame-pointer -fno-standalone-debug -gdwarf-4 -g2 -isysroot ../../../../../../Library/Developer/CommandLineTools/SDKs/MacOSX11.3.sdk -mmacosx-version-min=10.10.0 -ftrivial-auto-var-init=pattern -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wno-unknown-warning-option -Wno-unknown-pragmas -Wglobal-constructors -Wexit-time-destructors -Wshadow -Wexit-time-destructors -Wno-undefined-bool-conversion -Wno-tautological-undefined-compare -std=c++14 -fno-trigraphs -Wno-trigraphs -stdlib=libc++ -fno-exceptions -fno-rtti -fvisibility-inlines-hidden -x c++-header -c ../src/25ce732/build/precompile.h -o obj/base/base/precompile.h-cc.gch
/bin/sh: ../../../../../../usr/bin/clang++: No such file or directory
[42/412] ASM obj/base/base/call_with_eh_frame_asm.o
FAILED: obj/base/base/call_with_eh_frame_asm.o 
../../../../../../usr/bin/clang -MMD -MF obj/base/base/call_with_eh_frame_asm.o.d -DSYSTEM_NATIVE_UTF8 -D_LIBCPP_HAS_NO_ALIGNED_ALLOCATION -DCR_XCODE_VERSION= -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__ASSERT_MACROS_DEFINE_VERSIONS_WITHOUT_UNDERSCORES=0 -D_DEBUG -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DBASE_IMPLEMENTATION -I../src/25ce732 -Igen -fno-strict-aliasing -fstack-protector-strong -fcolor-diagnostics -fmerge-all-constants -fcrash-diagnostics-dir=../src/25ce732/tools/clang/crashreports -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -std=c11 -arch x86_64 -fno-standalone-debug -gdwarf-4 -g2 -isysroot ../../../../../../Library/Developer/CommandLineTools/SDKs/MacOSX11.3.sdk -mmacosx-version-min=10.10.0 -c ../src/25ce732/base/mac/call_with_eh_frame_asm.S -o obj/base/base/call_with_eh_frame_asm.o
/bin/sh: ../../../../../../usr/bin/clang: No such file or directory
[43/412] CXX obj/base/third_party/double_conversion/double_conversion/bignum.o
```
Any ideas?

**Comment by JackBoosY** [comment]
Ping @strega-nil-ms for review again.

**Comment by daschuer** [comment]
I have just rebased the commits from our agreement here https://github.com/microsoft/vcpkg/pull/21790#issuecomment-1022720986 to master. This allown fixes the CI failure.

**Comment by JackBoosY** [comment]
Sorry for late, can you please merge to master? I have no permission to push to your branch.

**Comment by daschuer** [comment]
Why? This still merges cleanly.

**Comment by JackBoosY** [comment]
@daschuer We should make sure the pipeline test will pass with the master branch.

**Comment by daschuer** [comment]
I have no interest to rebase this branch over and over again when there is a failing review. 

@strega-nil Can you have another look?

By the way: 
The maintainer edits checkbox is checked.

**Comment by JackBoosY** [comment]
@daschuer Very sorry for that.

**Review by strega-nil-ms — APPROVED** [review]

**Comment by JackBoosY** [comment]
The regression `libmikmod:x64-osx` is a baseline issue and will be fixed by other PR.

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the added support and for sticking with it for almost 6 months!

**Pull request merged**